### ISSUE TITLE
pusher.bind() not defined/supported

### ISSUE THREAD

**Issue by snaptopixel** [nikolalsvk/pusher-js-mock#23]
Thank you for building this lib, I'm trying to use it but `pusher.bind` does not seem to be supported.

https://pusher.com/docs/channels/using_channels/events#binding-on-the-client

**Comment by nikolalsvk** [comment]
Hey, @snaptopixel, thanks for opening this issue! Supporting `pusher.bind` is something we are interested in supporting.

Could you help us and explain how would you use such a feature? This will give an idea of how to create support for it, as well as prove some examples on how to use it.

**Comment by snaptopixel** [comment]
Sure, thanks @nikolalsvk, I think the example shown on the Pusher docs is a decent case, multiple channels with the same event-name can be bound once.

In our frontend, we initialize pages as web-components. Rather than require Pusher to be configured in each component, we are passing a pre-configured Pusher instance to them.

Ideally we could also pre-subscribe to channels so that our page component(s) only need to be aware of the event name.

In that case you can have something like:

```ts
componentDidLoad() {
  this.pusher.bind('some-event-here')
}
```
Rather than
```ts
componentDidLoad() {
  const ch = this.pusher.subscribe('some-channel')
  ch.bind('some-event-here')
}
```

**Comment by nikolalsvk** [comment]
Hey, @snaptopixel, I made some progress and added support for `pusher.bind`. Can you take a look at the [test I wrote](https://github.com/nikolalsvk/pusher-js-mock/blob/master/src/__tests__/pusher-js-mock.spec.ts#L56-L74) and see if this is something you would use?

Also, if you have any more suggestions or input, I'd really appreciate it.

**Comment by snaptopixel** [comment]
LGTM thanks much @nikolalsvk

**Comment by nikolalsvk** [comment]
Hey, @snaptopixel, I [just released 0.3.2](https://github.com/nikolalsvk/pusher-js-mock/releases/tag/0.3.2) so you can play with it if you want. Thanks for the idea :)
